assets/css/style.css

```diff
@@ -0,0 +1,559 @@
+@import "./reset.css";
+
+h4,
+h3,
+h2,
+h5,
+h6 {
+  font-weight: 600;
+  font-size: 16px;
+  cursor: pointer;
+}
+
+h2 {
+  padding: 10px;
+  color: grey;
+  cursor: text;
+}
+
+h5 {
+  color: #f72210;
+  margin-left: 15px;
+}
+
+a {
+  text-decoration: none;
+  color: black;
+}
+
+.china-img {
+  background-color: #f4f5f6;
+}
+
+.china-img:hover {
+  background-color: grey;
+}
+
+a:hover {
+  text-decoration: none;
+  color: black;
+}
+
+h4:hover {
+  text-decoration: underline;
+  text-decoration-color: red;
+}
+
+h6:hover {
+  color: #f72210;
+}
+
+body {
+  background-color: #f4f5f6;
+  color: #333;
+  font-size: 13px;
+  line-height: 18px;
+  font-family: Arial, Helvetica, sans-serif;
+  overflow-x: hidden;
+}
+
+header {
+  max-width: 1366px;
+  position: fixed;
+  top: 0;
+  z-index: 3;
+}
+
+article {
+  padding-top: 15px;
```

Comment:
You can use bootstrap classes for padding

Reply:
Now I used